lib/common/client/base.rb

```diff
@@ -0,0 +1,134 @@
+# frozen_string_literal: true
+
+require 'faraday'
+require 'common/client/errors'
+
+require 'common/exceptions/external/backend_service_exception'
+
+module Common
+  module Client
+    ##
+    # Raised if the Faraday adapter is a Faraday::Adapter::HTTPClient and the
+    # Common::Client::Middleware::Request::RemoveCookies middleware in not included.
+    #
+    class SecurityError < StandardError
+    end
+
+    ##
+    # Raised when the breakers Faraday middleware is not first in the stack.
+    #
+    class BreakersImplementationError < StandardError
```

Comment:
GIDS doesn't use `breakers`.
There are a few other `breakers` references in the PR too.

Reply:
Removing in https://github.com/department-of-veterans-affairs/gibct-data-service/pull/534